packages/node_modules/@ciscospark/internal-plugin-lyra/test/integration/spec/space.js

```diff
@@ -156,7 +156,7 @@ describe('plugin-lyra', function () {
         .then(() => spock.spark.internal.lyra.space.getCurrentBindings(lyraMachine.space))
         .then(({bindings}) => assert.lengthOf(bindings, 1)));
 
-      it('removes the binding between conversation and lyra space', () => spock.spark.internal.lyra.space.unbindConversation(lyraSpace, conversation)
+      skipInIE(it)('removes the binding between conversation and lyra space', () => spock.spark.internal.lyra.space.unbindConversation(lyraSpace, conversation)
```

Comment:
Could you add a note as to why this is skipped in IE?

Reply:
I will add a comment. This does not work on IE and we have no plans to support it.